bin/interactive-install.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+set -eu
+shopt -s extglob
+
+echo -e "\e[32mWelcome to the secureCodeBox!\e[0m"
+echo -e "This interactive installation script will guide you through all the relevant installation steps in order to have you ready to scan."
+
+echo -e "Start ? [y/n]"
+read -r line
+if [[ $line == *[Yy] ]]; then
+  echo -e "Starting!"
+else
+  exit
+fi
+
+echo
+echo -e "Checking kubectl.."
+kube=$(kubectl version)
+if [[ $kube == *"GitVersion"* ]]; then
+  echo -e "\e[32mIt's there!\e[0m"
+else
+  echo -e "\e[31mKubectl not found, quitting..\e[0m"
+  exit
+fi
+
+echo
+echo -e "Creating namespace securecodebox-system"
+kubectl create namespace securecodebox-system || echo -e "Namespace already exists.. "
+
+if [[ $PWD == *"bin" ]]; then
+  cd ..
+fi
+echo -e "Installing the operator in the securecodebox-system namespace"
+helm -n securecodebox-system upgrade --install securecodebox-operator ./operator/ || echo -e "\e[31mOperator installation failed, cancelling..e[0m"
+echo -e "\e[32mSuccessfully installed the operator!\e[0m"
+
+echo
+echo -e "Starting to install scanners.."
+cd scanners
+scanners=()
+for directory in */ ; do
```

Comment:
Here you should use an absolute path derived from the above mentioned base directory. Why? Here you completely rely that you are in the right directory, but due to edge cases you may end up somewhere else and slurp in everything listed in that location. So this may evolve to kinda code execution, when I achieve to trick this script iterating over a directory where I put my malicious helm charts.